solutions/devsprint-bruno-ramos-5/GitHubApp/Screens/List/Cell/RepositoryViewCell.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  RepositoryViewCell.swift
+//  GitHubApp
+//
+//  Created by Felipe on 06/07/22.
+//
+
+import UIKit
+
+struct RepositoryViewCellConfiguration{
+    let repositoryName: String
+    let repsitoryOwnerName: String
+}
+
+
+class RepositoryViewCell: UITableViewCell {
+    
+    private lazy var containerVS: UIStackView = {
+        let stack = UIStackView()
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        stack.axis = .vertical
+        stack.spacing = 6
+        
+        return stack
+    }()
+    
+    lazy var repositoryNameLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.systemFont(ofSize: 18)
+        
+        return label
+    }()
+    
+    lazy var repositoryOwnerNameLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .gray
+        label.font = UIFont.systemFont(ofSize: 14)
+        
+        return label
+    }()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        self.accessoryType = .disclosureIndicator
+        self.configureSubview()
+        self.setUpConstraints()
```

Comment:
O que acha de usar o protocolo ViewCode na Cell? Assim você precisaria chamar apenas o método setup do template method, assim ficaria apenas:
```suggestion
        setup()
```

Reply:
Alterado